docs/data/charts/pyramid-demo/populationByEducationLevel.ts

```diff
@@ -0,0 +1,95 @@
+// Data by Wittgenstein Centre (2024) – processed by Our World in Data
+// Derived from https://ourworldindata.org/grapher/world-population-level-education?time=2020&v=1&csvType=filtered&useColumnShortNames=false
+// Year 2020
+
+const fullData = {
```

Comment:
Does this file needs to be duplicated in funnel and here? We could use the same dataset